packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -200,8 +232,19 @@ export default {
         };
     },
 
+    watch: {
+        immediate: true,
+        isAuthFinished () {
+            if (this.isAuthFinished) {
+                this.initialise();
+            }
+        }
+    },
```

Comment:
I now think it should be done like this ... (also unsure why it showed up as a change as the same as upstream .... ummm)

```suggestion
    watch: {
        isAuthFinished: {
            immediate: true,
            async handler (value) {
                if (value) {
                    await this.initialise();
                }
            }
        }
    },
```